ce/api/percentileanomaly.py

```diff
@@ -183,62 +297,91 @@ def canonical_timestamp(timestamp, timescale, timeidx):
                                                             row['units']))
 
                     if row['climatology'] == climatology:
-                        add_to_nested_dict(projected_data,
-                                           row["mean"],
-                                           row["timescale"],
-                                           ctimestamp,
-                                           row["model"])
+
+                        projected_data, model_counter = add_to_nested_li(projected_data, row, ctimestamp, model_counter)
+
                     elif (calculate_anomaly and
                             row['model'] == baseline_model and
                             row['climatology'] == baseline_climatology):
-                        add_to_nested_dict(baseline_data, row['mean'],
-                                           row['timescale'], ctimestamp)
+
+                        baseline_data = add_to_nested_li(baseline_data, row, ctimestamp)
```

Comment:
To receive the uniform result of this, use `baseline_data, _ = ...`. The `_` is a common idiom for "thing I don't care about."